services/frontend-service/src/legacy-ui/App/index.tsx

```diff
@@ -51,7 +52,8 @@ export const Spinner: React.FC<any> = (props: any) => {
 };
 
 const GetOverview = (props: { children: (r: api.GetOverviewResponse) => JSX.Element }): JSX.Element | null => {
-    const getOverview = React.useCallback((api) => api.overviewService().StreamOverview({}), []);
+    const { authHeader } = React.useContext(AuthTokenContext);
```

Comment:
```suggestion
    const getOverview = React.useCallback((api,authHeader) => api.overviewService().StreamOverview({}, authHeader), []);
```

Reply:
Same goes for `useUnaryCallback`, 
in `useUnaryCallback` & `useObservable` add this 
```
const { authHeader } = React.useContext(AuthTokenContext);
```
then use it like this:
```
callback(api, authHeader)
```
then please delete all the const { authHeader } = React.useContext(AuthTokenContext); from the requests

Reply:
seems like indirection for its own sake, but sure

Reply:
> 



> seems like indirection for its own sake, but sure

I didn’t get this, please can you clarify ?

Reply:
Needed additional type casting because of browser headers being required

Reply:
https://github.com/freiheit-com/kuberpult/pull/285/commits/cdde5751fb07ac239868e0f1af5485197576ce40